### ISSUE TITLE
Masthead: Hamburger needs < Back eyebrow

### ISSUE THREAD

**Issue by larahanlon2** [carbon-design-system/ibm-dotcom-library#143]
When a user clicks one level down into the hamburger menu a < Back eyebrow should appear to allow them to navigate back to the "parent" menu.

Currently, there is no way for me to go back to the first level:
<img width="560" alt="Screen Shot 2019-10-02 at 11 45 41 AM" src="https://user-images.githubusercontent.com/53093550/66059719-7eb90880-e50a-11e9-9c2d-ea3677789574.png">
<img width="575" alt="Screen Shot 2019-10-02 at 11 42 21 AM" src="https://user-images.githubusercontent.com/53093550/66059725-7fea3580-e50a-11e9-8f65-1bca2208cba2.png">

**Actual view should be:**
<img width="421" alt="Screen Shot 2019-10-02 at 11 48 19 AM" src="https://user-images.githubusercontent.com/53093550/66059787-9b554080-e50a-11e9-8903-aa27154ba5e6.png">
